apps/antalmanac/package.json

```
@@ -18,7 +18,8 @@
         "deploy": "gh-pages -d build",
         "format": "prettier --write src",
         "lint": "eslint src",
-        "test": "vitest"
+        "test:watch": "vitest",
+        "test": "vitest run"
```

I think this is sufficient.

```suggestion
        "test": "vitest"
```

---

`vitest` runs interactively by default so if we have it in our CI then it'll block everything and never finish. This is what the Turbo handbook recommended.